libs/ngxtension/gestures/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './drag';
+export * from './zoneless-gesture';
```

Comment:
personal opinion: I always prefer to explicitly export what I want to export

export {xxx} from '....'

Reply:
I do it on a case-by-case basis. There's no value to do `export { xxx }` in these cases